tensorboard/plugins/graph/graphs_plugin_v2_test.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Integration tests for the Graphs Plugin for TensorFlow v2."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+import os.path
+
+from google.protobuf import text_format
+import tensorflow as tf
+
+from tensorboard.compat.proto import graph_pb2
+from tensorboard.plugins.graph import graphs_plugin_test
+
+
+tf.enable_v2_behavior()
```

Comment:
Is this line necessary?  If this test will pass under TF 1.x as well that's ideal, because then when we set up tests against TF 1.x and 2.0 the tests will run using the default behavior in both cases (as opposed to being forced into 2.0 behavior in both).

Reply:
Argh, I realized why Keras was not writing the conceptual graph. It is implemented in callback_v2 and I am not sure doing `enable_v2_behavior` will load that symbol. Will report back what I can do.

Reply:
Can you just refer to `tf.compat.v2.keras.callbacks.TensorBoard` instead?

Reply:
Doh, I am quite dumb. I did check the source of callbacks.py in the virtualenv and confirmed that the new code did not land in the tf-nightly.